postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -180,6 +180,10 @@ def __init__(self, check, config, shutdown_callback):
         )
         if collection_interval <= 0:
             collection_interval = DEFAULT_COLLECTION_INTERVAL
+        
+        # if regular samples is disabled, only need to collect as often as activity is sampled
+        if is_affirmative(config.statement_samples_config.get('enabled', False)):
```

Comment:
should this be:

```
        if not is_affirmative(config.statement_samples_config.get('enabled', True)):
```

? the feature should be enabled by default

Reply:
oh, right. thank you!
I got confused, thought is_affirmative was checking the get() result and seeing it's equal to a second parameter, but now I see get() takes the second parameter as a default and is_affirmative just produces a boolean from it 👍